docs/network/interfaces_and_networks.md

```diff
@@ -47,40 +47,51 @@ fields:
 </tr>
 <tr class="even">
 <td><p><code>multus</code></p></td>
-<td><p>Secondary network provided using Multus</p></td>
+<td><p>Secondary network provided using Multus<br>
+or Primary network when Multus is defined as default</p></td>
 </tr>
 </tbody>
 </table>
 
 ### pod
 
-A `pod` network represents the default pod `eth0` interface configured
+Represents the default pod interface (typically `eth0`) configured
 by cluster network solution that is present in each pod.
+Sine this network is native to kubernetes it has some advantages over external 
+networks types (currently only `multus`)
+- Support of Kueberentes [EndpointSlices](https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/),
+[Services](https://kubernetes.io/docs/concepts/services-networking/service/) and the functionality that utilizes them 
+both in kubernetes and the echo system (e.g. [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) ) 
+- VM access to Kubernetes API server 
```

Comment:
I would avoid mentioning specific k8s APIs/features they as they can change and it lack some context how to use it with VMs.

How about?
```suggestion
Main advantage of connecting VMs to cluster pod network is having access to the cluster services, such as IP addressing, routes, DNS, access other workloads and Services.
```

Reply:
@orelmisan suggested that I mention `services` and  I think that the APIs that are mentioned are GA and will probably outlive kubevirt :)

Reply:
@orelmisan I've accepted your suggestion